blobmodel/stochasticality.py

```diff
@@ -115,6 +123,11 @@ def sample_blobs(
         wys = self._draw_random_variables(self.wy_dist, self.wy_parameter, num_blobs)
         vxs = self._draw_random_variables(self.vx_dist, self.vx_parameter, num_blobs)
         vys = self._draw_random_variables(self.vy_dist, self.vy_parameter, num_blobs)
+        spxs = self._draw_random_variables(self.spx_dist, self.spx_parameter, num_blobs)
+        spys = self._draw_random_variables(self.spy_dist, self.spy_parameter, num_blobs)
+        # For now, only a lambda parameter is implemented
+        spxs = [{"lam": s} for s in spxs]
+        spys = [{"lam": s} for s in spys]
```

Comment:
Out of interest: We don't intend to use varying spxs in the future, right?

Reply:
You mean distributed pulse parameters? We do! I had already worked with that in the fppy code to test some of the 1d time delay estimation techniques.
Ill mark this as resolved for now since it doesn't have to code to do, feel free to reopen it.